src/axom/quest/readers/ProEReader.hpp

```diff
@@ -73,6 +82,32 @@ class ProEReader
    */
   virtual int read();
 
+  /*!
+   * \brief Sets the bounding box to include nodes.
+   * 
+   * When reading in ProE tets, include only those tets that are fully
+   * contained in the bounding box.  If the box is left unset, or if set
+   * to an invalid box, all tets are included.
+   */
+  virtual void setBoundingBox(BBox3D& box);
+
+  /*!
+   * \brief Sets the bounding box to include nodes.
+   * 
+   * When reading in ProE tets, include only those tets that have at least 
+   * one node in the bounding box.  If the box is left unset, or if set
+   * to an invalid box, all tets are included.
+   */
+  virtual void setInclusiveBoundingBox(BBox3D& box);
```

Comment:
API Design: These functions are a bit confusing to me since they set the bounding box for the domain _and also_ set the lambda/predicate used for filtering the tetrahedra to either include tets that have all four vertices within the bounding box, or those with at least vertex inside.

It's not clear from the documentation that a user needs to call one of `setBoundingBox()` or `setInclusiveBoundingBox()` or `setTetPred()` and that the default would effectively be to include all tetrahedra.

Suggestion: Please consider the following possible improvements:
* Put all these functions in a doxygen group that describes that one of these should be called and what the default is
  (or describe this in the class doxygen comment)
* Replace the `setBoundingBox()` and `setInclusiveBoundingBox()` functions with a single function called something like `setTetPredFromBoundingBox(BBox3D& box, bool inclusive)` where the second parameter controls whether one or all vertices need to be inside.

Reply:
I'll put any predicate functions in a doxygen group and describe bounding box action in the class comment.  Thanks--- that's a good idea.

Reply:
Second bullet: OK, I agree with this too.  It will be a simpler API: one fewer function with a more descriptive name.

Reply:
Combined `setBoundingBox()` and `setInclusiveBoundingBox()` into `setTetPredFromBoundingBox()` as discussed.  Used Doxygen grouping to clarify relationship between `setTetPred()` and `setTetPredFromBoundingBox()`.  Expanded Doxygen generally.